### ISSUE TITLE
Exposing Search Menu to Non-Admin Users

### ISSUE THREAD

**Issue by ChristopherKB** [Graylog2/graylog2-server#8921]
## Expected Behavior

Read-only Users should be able to see the Search Menu item in their UI. 
## Current Behavior

Read-only users can see only the Streams menu item and must know to click on it to get to the search page.
## Possible Solution

I believe this is just a holdover from a time when we could not restrict the search page by stream. Now that the new search integrates limitation by streams, there is no longer any reason to hide this menu item. Konrad called this a bug, but it may simply be a holdover that nobody has looked at recently.

## Steps to Reproduce (for bugs)

1. Create user with default Reader role
2. Log in and observe the menu.
3.
4.

## Context

Read only users are confused when they can't see something called Search. The word streams does not intuitively lead them to a search page. At least two customers have expressed this concern recently.
## Your Environment

* Graylog Version: 3.x and up
* Java Version:
* Elasticsearch Version:
* MongoDB Version:
* Operating System:
* Browser version:

**Comment by bernd** [comment]
Duplicate of #8745 and #8779. This will be fixed once #8917 is merged.
